docs/open-source/react-quickstart.mdx

```diff
@@ -57,34 +57,22 @@ uvicorn main:app --port 3000
 You now have a server with a Vocode websocket route at localhost:3000! You can now use the `useConversation` hook with your self-hosted backend as follows:
 
 ```javascript
-const { status, start, stop, analyserNode } = useConversation({
+const { status, start, stop, error, analyserNode } = useConversation({
   backendUrl: "<YOUR_BACKEND_URL>", // looks like ws://localhost:3000/conversation or wss://asdf1234.ngrok.app/conversation if using ngrok
   audioDeviceConfig: {},
 });
 ```
 
-# Demo installation and setup
+Use the `status`, `start`, and `stop` objects within your React components to control conversations with your self-hosted backend, e.g.
 
-Clone the `vocode-react-demo` [repository](https://github.com/vocodedev/vocode-react-demo).
-
-```
-$ git clone https://github.com/vocodedev/vocode-react-demo.git
-```
-
-Run npm install inside the directory to download all of the dependencies.
-
-```
-$ npm install
 ```
+<>
+{status === "idle" && <p>Press me to talk!</p>}
+{status == "error" && error && <p>{error.message}</p>}
 
-Set your Client SDK key inside of your `.env`
-
-```
-REACT_APP_VOCODE_API_KEY=YOUR KEY HERE
-```
-
-Start the application
-
-```
-$ npm start
+<button
+  disabled={["connecting"].includes(status)}
```

Comment:
what is "connecting" here?

Reply:
it's an interim status exposed by the react-sdk: https://github.com/vocodedev/vocode-react-sdk/blob/main/src/types/conversation.ts#L6

i'm going to add something here that shows how to install the react-sdk